### DISCUSSION TITLE
打包出错了，求助

### DISCUSSION THREAD

**Discussion by midpoint** [tw93/Pake#246] [Q&A]
pake https://www.baidu.com
To build the Linux app, you need to install Rust and Linux package
See more in https://tauri.app/v1/guides/getting-started/prerequisites#installing

✔ please input your application name … baidu
You have not provided an app icon, use the default icon.(use --icon option to assign an icon)

up to date in 4m

77 packages are looking for funding
  run `npm fund` for details

> pake-cli@1.2.1 build
> npm run tauri build --release

> pake-cli@1.2.1 tauri
> tauri build

   Compiling app v0.1.0 (/home/mj/.node/lib/node_modules/pake-cli/src-tauri)
    Finished release [optimized] target(s) in 15.19s
    Bundling baidu_1.0.0_amd64.deb (/home/mj/.node/lib/node_modules/pake-cli/src-tauri/target/release/bundle/deb/baidu_1.0.0_amd64.deb)
    Bundling baidu_1.0.0_amd64.AppImage (/home/mj/.node/lib/node_modules/pake-cli/src-tauri/target/release/bundle/appimage/baidu_1.0.0_amd64.AppImage)
       Error failed to bundle project: error running appimage.sh: error running appimage.sh: `failed to run /home/mj/.node/lib/node_modules/pake-cli/src-tauri/target/release/bundle/appimage/build_appimage.sh`
file:///home/mj/.node/lib/node_modules/pake-cli/dist/cli.js:1803
                reject(new Error(`${code}`));
                       ^

Error: 1
    at file:///home/mj/.node/lib/node_modules/pake-cli/dist/cli.js:1803:24
    at /home/mj/.node/lib/node_modules/pake-cli/node_modules/shelljs/src/exec.js:145:9
    at ChildProcess.exithandler (node:child_process:420:5)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)

**Comment by cnmeeia** [comment]
貌似 看看是 环境 没有 安装好

**Reply by midpoint** [reply]
能指点一下是什么没安装吗？

**Reply by cnmeeia** [reply]
To build the Linux app, you need to install Rust and Linux package
See more in https://tauri.app/v1/guides/getting-started/prerequisites#installing

Rust
